fife/__main__.py

```diff
@@ -52,6 +52,14 @@ def main():
     )
     print(f"Data processing time: {time() - checkpoint_time} seconds")
 
+    by_feature = config.get("BY_FEATURE", None)
+    if by_feature == "None":
```

Comment:
"None" is a reasonable default value, but for conformity with the rest of the API let's use the empty string "" as the default value instead.